website/content/docs/custom-widgets.md

```diff
@@ -198,3 +202,92 @@ You can also return a promise from `isValid`. While the promise is pending, the
 ```
 
 **Note:** Do not create a promise inside `isValid` - `isValid` is called right before trying to persist. This means that even if a previous promise was already resolved, when the user hits 'save', `isValid` will be called again. If it returns a new promise, it will be immediately marked as "in error" until the new promise resolves.
+
+## Creating a custom widget using JSX syntax with a build step
```

Comment:
We can rephrase it to `Writing custom widgets as a separate package` as mentioned before.